src/Instrumentation/ReactHTTP/examples/http_only.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+declare(strict_types=1);
+
+use OpenTelemetry\API\Common\Time\Clock;
+use OpenTelemetry\API\Trace\Propagation\TraceContextPropagator;
+use OpenTelemetry\SDK\Common\Export\Stream\StreamTransportFactory;
+use OpenTelemetry\SDK\Resource\ResourceInfoFactory;
+use OpenTelemetry\SDK\Sdk;
+use OpenTelemetry\SDK\Trace\Sampler\AlwaysOnSampler;
+use OpenTelemetry\SDK\Trace\SpanExporter\ConsoleSpanExporter;
+use OpenTelemetry\SDK\Trace\SpanProcessor\BatchSpanProcessor;
+use OpenTelemetry\SDK\Trace\TracerProvider;
+use Psr\Http\Message\ResponseInterface;
+use React\EventLoop\Loop;
+use React\Http\Browser;
+use React\Http\Message\Request;
+
+require_once dirname(__DIR__) . '/vendor/autoload.php';
+
+$transport = (new StreamTransportFactory())->create('php://output', 'application/json');
+$exporter = new ConsoleSpanExporter($transport);
+
+$tracerProvider =  new TracerProvider(
+    new BatchSpanProcessor($exporter, Clock::getDefault()),
+    new AlwaysOnSampler(),
+    ResourceInfoFactory::emptyResource(),
+);
+
+Sdk::builder()
+    ->setTracerProvider($tracerProvider)
+    ->setPropagator(TraceContextPropagator::getInstance())
+    ->setAutoShutdown(true)
+    ->buildAndRegisterGlobal();
+
+$root = $tracerProvider->getTracer('react-http-demo')->spanBuilder('root')->startSpan();
+
+Loop::futureTick(static function () use ($root) {
```

Comment:
Is the `Loop::futureTick()` needed?

Reply:
Nope, you're right. I dealt with that elsewhere. Removing in the next commit.